virtual_rainforest/core/readers.py

```diff
@@ -0,0 +1,120 @@
+"""API documentation for the :mod:`core.readers` module.
```

Comment:
As discussed, I'd drop this file and just have one reader for now.

Reply:
I am pretty sure that I am going to want to use CSV files for the plant module - simply easier to create and edit. I could convert that to NetCDF but there are going to be several files where we want to be able to tinker quickly and easily with input values to see what happens.